javav2/example_code/acmpca/src/main/java/com/example/acmpca/GetCertificate.java

```diff
@@ -0,0 +1,78 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.example.acmpca;
+
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.acmpca.AcmPcaClient;
+import software.amazon.awssdk.services.acmpca.model.AcmPcaException;
+import software.amazon.awssdk.services.acmpca.model.GetCertificateRequest;
+import software.amazon.awssdk.services.acmpca.model.GetCertificateResponse;
+import software.amazon.awssdk.services.acmpca.waiters.AcmPcaWaiter;
+
+// snippet-start:[acmpca.java2.GetCertificate.main]
+/**
+ * Before running this Java V2 code example, set up your development environment, including your
+ * credentials.
+ *
+ * <p>For more information, see the following documentation topic:
+ *
+ * <p>https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
+ */
+public class GetCertificate {
+
+  public static void main(String[] args) throws Exception {
+
+    final String usage =
+        """
```

Comment:
repeating my earlier comments about consistency when it comes to the formatting of these usage strings. both indentation and newline (or lack thereof)